### DISCUSSION TITLE
[Informationssystem] Wetter anzeigen Lehrer / Schüler

### DISCUSSION THREAD

**Discussion by two-Gee** [SE-TINF22B2/G2-Schoolify#273] [User Stories]
### User Story 
Als Lehrer / Schüler möchte ich dass mir das Wetter angezeigt wird, damit ich ich mich optimal auf den Schulweg vorbereiten kann. 

### Beschreibung
In der Navigationsleiste soll das Wetter kompakt mit Temperatur und Kurzbeschreibung (z.B sonnig) angezeigt werden. 
Falls man auf die kompakte Anzeige klickt, soll eine detaillierte Ansicht des Wetter am heutigen Tag gezeigt werden.
 
### Akzeptanz Kriterien
- [ ]  Kompakte Anzeige des Wetters in Navigationsleiste 
- [ ]  Detaillierte Anzeige

### Issues
Frontend:
- #302 
- #303

Backend:
- #304
- #305

**Comment by tilwbr** [comment]
Mögliche API: https://www.weatherapi.com/
